core/iwasm/compilation/aot.h

```diff
@@ -19,6 +19,15 @@ extern "C" {
 #define AOT_FUNC_PREFIX "aot_func#"
 #endif
 
+#ifndef AOT_FUNC_PREFIX2
+#define AOT_FUNC_PREFIX2 "aot_func2#"
```

Comment:
"aot_func2#n" is very similar to "aot_func#n", it makes a little confusing to read the llvm-ir file and the generated machine code. Could we change to other name, how about:
```C
#define AOT_FUNC_INTERNAL_PREFIX "aot_func_internal#"
```
let `aot_func#n` call `aot_func_interanl#n`?

Reply:
ok